### PR TITLE
Add support of ANSI escape sequence for clear line

### PR DESCRIPTION
**Pull request by noborus**
ov(and other pagers) cannot be erased.
Actually, set the style from the end of the line.
This solves #650 .

### CODE REVIEW

File: oviewer/convert_es.go [hunk 1]
```diff
@@ -75,15 +75,19 @@ func (es *escapeSequence) convert(st *parseState) bool {
...
+		case mainc == 'm':
 			st.style = csToStyle(st.style, es.parameter.String())
-		} else if mainc >= 'A' && mainc <= 'T' {
-			// Ignore.
-		} else {
-			if mainc >= 0x30 && mainc <= 0x3f {
-				es.parameter.WriteRune(mainc)
-				return true
+		case mainc == 'K':
+			if es.parameter.String() != "1" {
```
Comment (KapJI) [inline oviewer/convert_es.go:82]: I don't think this is correct. This basically implements `ESC[K` and `ESC[K0` but `ESC[K1` erases from cursor to the beginning of line. And `ESC[K2` erases the whole line.

Comment (noborus) [reply oviewer/convert_es.go:82]: Thank you for your comment.
I didn't understand, what do you think is the best way to do this?

Comment (KapJI) [reply oviewer/convert_es.go:82]: I mean that at least it shouldn't accept `ESC[K2`. But ideally implementing it and `ESC[K1`  properly. There are also `ESCj` commands which move cursor and probably worth to be supported as well.

Comment (noborus) [reply oviewer/convert_es.go:82]: So does that mean I only need to style spaces and 0's here?
The pager won't clear the lines in 2, but the styling will work similarly.
es

Comment (KapJI) [reply oviewer/convert_es.go:82]: You screenshot shows how it should work. Except for b`3K` which doesn't seem valid value.

So minimally we need `K` and `0K`.

But ideally support `1K` and `2K` as well.

Comment (noborus) [reply oviewer/convert_es.go:82]: > So minimally we need K and 0K.

I got it. Work only with K and 0k.

> But ideally support 1K and 2K as well.

It is still unknown whether it is more correct to work as a pager.

Thank you.

**Pull request merged**